doc/manual/src/language/syntax.md

```diff
@@ -249,20 +249,18 @@ Elements in a list can be accessed using [`builtins.elemAt`](./builtins.md#built
 
 An attribute set is a collection of name-value-pairs (called *attributes*) enclosed in curly brackets (`{ }`).
 
-An attribute name can be an identifier or a [string](#string).
-An identifier must start with a letter (`a-z`, `A-Z`) or underscore (`_`), and can otherwise contain letters (`a-z`, `A-Z`), numbers (`0-9`), underscores (`_`), apostrophes (`'`), or dashes (`-`).
+An attribute *name* can be an [identifier](./identifiers.md) or a [string](#string-literal).
```

Comment:
```suggestion
An attribute *name* can be written as an [identifier](./identifiers.md) or a [string](#string-literal).
```

The name itself is always a string; there is no difference between `{ "a" = 1; }` and `{ a = 1; }`.